WowUp.WPF/Services/AddonService.cs

```diff
@@ -175,10 +176,10 @@ public async Task<List<Addon>> GetAddons(WowClientType clientType, bool rescan =
                 {
                     var newAddons = await ScanAddons(clientType);
                     addons = UpdateAddons(addons, newAddons);
+                    AddonListUpdated?.Invoke(this, new EventArgs());
```

Comment:
You can use EventArgs.Empty() here.  Very minor, but it's one less object initialization.

Reply:
thank you for the insight, changed it.